src/config.rs

```diff
@@ -77,7 +77,7 @@ fn default_mergetool_config() -> config::Config {
     // loaded by separate builder.
     config::Config::builder()
         .add_source(config::File::from_str(
-            include_str!("merge-tools.toml"),
+            include_str!("merge_tool_config.toml"),
```

Comment:
Once we have a few more files, maybe we should move them into a `src/config/` directory. (No action required.)